pkg/kubelet/volumemanager/populator/desired_state_of_world_populator.go

```diff
@@ -325,6 +325,7 @@ func (dswp *desiredStateOfWorldPopulator) processPodVolumes(
 				err)
 			dswp.desiredStateOfWorld.AddErrorToPod(uniquePodName, err.Error())
 			allVolumesAdded = false
+			continue
 		}
 
 		klog.V(4).Infof(
```

Comment:
Instead of `continue` we could also use an `else` block... not sure which one is more readable (and don't have a strong preference myself) but just wanted to toss out the additional option.

Reply:
In the code I have seen so far in k8s, err != nil block is normally by itself.

There are two more statements following the block - hence an else doesn't seem to convey better readability.